api/src/controllers/files.ts

```diff
@@ -13,6 +12,9 @@ import { FilesService, MetaService } from '../services';
 import { File, PrimaryKey } from '../types';
 import asyncHandler from '../utils/async-handler';
 
+// @ts-ignore
+import formatTitle from '@directus/format-title';
+
```

Comment:
What was it complaining about here?

Reply:
Same typing thing:

```
src/database/migrations/run.ts:8:25 - error TS7016: Could not find a declaration file for module '@directus/format-title'. '/Users/rijkvanzanten/Developer/directus/directus/node_modules/.pnpm/@directus+format-title@9.15.0/node_modules/@directus/format-title/dist/format-title.cjs.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/directus__format-title` if it exists or add a new declaration (.d.ts) file containing `declare module '@directus/format-title';`

8 import formatTitle from '@directus/format-title';
                          ~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
I see, thx! 👍 WDYT about updating `@directus/format-title` to v10 and importing it dynamically as well? It seems like it's being used in async contexts anyway.